File: LeetCode/TwoSum/python/TwoSum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        length = len(nums)
        ind = 0
        for i in nums:
            val = target-i
            if(val in nums[ind+1:]):
                return [ind, nums.index(val, ind+1)]
            ind += 1
                
        return []
        

File: LeetCode/TwoSum/java/TwoSum.java
class TwoSum {
    public int[] twoSum(int[] nums, int target) {
        Map<Integer, Integer> num_map = new HashMap<>();
        
        for(int i=0; i<nums.length; i++){
            if(num_map.containsKey(target-nums[i])){
                return new int[] {num_map.get(target-nums[i]), i};
            }
            num_map.put(nums[i], i);
        }
        return null;
        
    }
}
